lib/rouge/lexers/svelte.rb

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class Svelte < HTML
+      desc 'Svelte single-file components (https://svelte.dev/)'
+      tag 'svelte'
+      filenames '*.svelte'
+
+      # are these accurate?
+      mimetypes 'text/x-svelte', 'application/x-svelte'
```

Comment:
This helps to hint Rouge guesser to select the correct lexer. As far as I know, there are no official MIME Types for Svelte as per [IANA media types] (https://www.iana.org/assignments/media-types/media-types.xhtml#application). There is no harm leaving here so if any server would like to send this information through.

```suggestion
      mimetypes 'text/x-svelte', 'application/x-svelte'
```

Reply:
For additional context, we can verify `mimetype` guessing via the CLI.

```shell
./bin/rougify hightlight --mimetype "text/x-svelte" <file_name_without_extension>
```